src/Chainweb/Transaction.hs

```diff
@@ -121,6 +122,8 @@ creationTimeOf :: forall c . Command (Payload PublicMeta c) -> TxCreationTime
 creationTimeOf = _pmCreationTime . _pMeta . _cmdPayload
 {-# INLINE creationTimeOf #-}
 
+payloadWithText :: Command PayloadWithText -> PayloadWithText
+payloadWithText = _cmdPayload
```

Comment:
i don't think this passes the Fairbarn threshold - it's just an accessor

Reply:
see latest commit